scene/main/viewport.cpp

```diff
@@ -784,19 +799,20 @@ void Viewport::update_canvas_items() {
 	_update_canvas_items(this);
 }
 
-void Viewport::_set_size(const Size2i &p_size, const Size2i &p_size_2d_override, const Rect2i &p_to_screen_rect, const Transform2D &p_stretch_transform, bool p_allocated) {
-	if (size == p_size && size_allocated == p_allocated && stretch_transform == p_stretch_transform && p_size_2d_override == size_2d_override && to_screen_rect == p_to_screen_rect) {
+void Viewport::_set_size(const Size2i &p_size, const Size2i &p_size_2d_override, const Rect2i &p_to_screen_rect, const Transform2D &p_stretch_transform, bool p_allocated, uint32_t p_view_count, bool p_force_resize) {
+	if (!p_force_resize && size == p_size && view_count == p_view_count && size_allocated == p_allocated && stretch_transform == p_stretch_transform && p_size_2d_override == size_2d_override && to_screen_rect == p_to_screen_rect) {
```

Comment:
I don't think the force is needed anymore now that we include the view_count

Reply:
Well, the reason I wanted to keep it was for smartphone AR where the size and view count may actually stay the same, but in a world where we have external textures provided by the XR interface, we'd want the render target to get remade to use those. However, since we don't have the external textures thing yet, I'll just remove it for now.